### ISSUE TITLE
[Suggestion] Improved selection recognition

### ISSUE THREAD

**Issue by johndelker** [Nettakrim/Spyglass-Astronomy#43]
When looking through a spyglass at a constellation, it only shows the name of the constellation if you look at one of its stars. It would be great if the name showed when looking at any line as well.

**Comment by Nettakrim** [comment]
yeah, the main issue is that itd be sorta hard to calculate that lol
